xclim/ensembles/_robustness.py

```diff
@@ -323,7 +331,12 @@ def mwu_wrapper(f, r):  # This specific test can't manage an all-NaN slice
             changed = abs(delta / ref.mean("time")) > kwargs["rel_thresh"]
         else:
             raise ValueError("Invalid argument combination for test='threshold'.")
-
+    elif test == "ipcc-advanced":
+        # Detrend ref
+        refy = ref.resample(time="YS").mean()
+        ref_detrended = detrend(refy, dim="time", deg=1)
+        gamma = np.sqrt(2 / 20) * 1.645 * ref_detrended.std("time")
```

Comment:
In the NB, you say we can provide a custom gamma, but I don't see it here.

Reply:
See other comment, but indeed, I'll make the distinction clearer in the notebook.